src/com/nutrons/steamworks/Shooter.java

```diff
@@ -41,29 +43,46 @@ public Shooter(LoopSpeedController shooterController, Flowable<Boolean> shooterB
     this.shooterButton = shooterButton;
     this.distance = distance;
     this.setpointHint = setpointHint;
-    this.variableSetpoint = this.distance.filter(x -> x != 0.0).map(x -> 11.778 * x + 1858.7).share();
```

Comment:
why are these changes not in unbag-testing? are they on a different branch? I thought we used unbag testing in Boston, in which case unbag testing would have the correct values? I want to make sure these tuning commits belong here...